<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartNodeDialog</class>
 <widget class="QDialog" name="StartNodeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Start node</string>
  </property>
  <widget class="QLabel" name="lblHost">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Host:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboHost">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>10</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="lblProgram">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Program:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLaunch">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>267</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblArguments">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>69</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arguments:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editArguments">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>70</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboProgram">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>40</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>267</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
